torch/_decomp/decompositions.py

```diff
@@ -1218,3 +1218,60 @@ def stack(tensors: List[Tensor], dim: int = 0) -> Tensor:
         return out.view(result_sizes)
     else:
         return torch.cat(get_stack_inputs(tensors, wrapped_dim), dim)
+
+
+@register_decomposition(aten.frac)
```

Comment:
We should probably start adding the `.default` in anticipation of Ed's changes in https://github.com/pytorch/pytorch/pull/76835.

Reply:
What's the rule for determining when .default is needed?